doc/source/science_guide/sg_boundary_forcing.rst

```diff
@@ -323,6 +323,8 @@ where
    \end{aligned}
    :label: dragio-defs
 
+This is done under if the namelist option ``calc_dragio`` is set to true.
```

Comment:
Need to fix the grammar in this sentence.  Actually, I recommend moving it up to line 310:  
"When the namelist option ``calc_dragio`` is set to true, the ice-ocean drag coefficient... is computed from ..."

Reply:
OK, good suggestion, thanks.